profiles/source/Source_FPD.yaml

```diff
@@ -11,8 +11,7 @@ dsFPD:
       MIN_BRIGHTNESS : 0              # Minimum brightness value of FPD LEDs
       DEFAULT_LEVELS : 1              # Default Brightness level for the FP text display
       MAX_CYCLERATE : 2               # Maximum Rate at which LED is rotating during scrolling
-      DEFAULT_COLOR_MODE : 0          # Color Mode of LED (Single(0) or Multicolored(1))
-      MULTI_COLOR_MODE : 2
+      DEFAULT_COLOR_MODE : 1          # Color Mode of LED (Single(0) or Multicolored(1))
       supportedColors :
```

Comment:
Can you please add comments as present in the videoport yaml files

example:
#dsFPD_COLOR_BLUE   dsFPDColor_Make(0, 0, 0xFF)          ///< Blue color LED                 #
#dsFPD_COLOR_GREEN ///< Green color LED        
#dsFPD_COLOR_RED     ///< Red color LED                 
#dsFPD_COLOR_YELLOW  ///< Yellow color LED               
#dsFPD_COLOR_ORANGE  ///< Orange color LED               
#dsFPD_COLOR_WHITE   ///< White color LED       
      supportedColors :
      - dsFPD_COLOR_GREEN
      - dsFPD_COLOR_RED
      - dsFPD_COLOR_ORANGE
      - dsFPD_COLOR_WHITE

Please consider for both source and sink yaml. Also consider for the Indicator_Type ,  SupportedLEDStates